src/Microsoft.DotNet.Build.Tasks.Feed/PackageFiles/Microsoft.DotNet.Build.Tasks.Feed.targets

```diff
@@ -125,15 +125,15 @@
       <ManifestFile Condition="'$(ManifestFile)' == ''">$(ManifestTempDir)build.xml</ManifestFile>
     </PropertyGroup>
 
-    <ParseFeedUrl FeedUrl="$(ExpectedFeedUrl)">
+    <ParseBlobUrl BlobUrl="$(ExpectedFeedUrl.Replace('/index.json', '')">
```

Comment:
I was working on merging this into my changes and noticed the parens are unbalanced here.